opc_ua/opcua_client_maker.py

```diff
@@ -132,6 +132,9 @@ def OnDrop(self, x, y):
         self.ParentWindow.OnNodeDnD()
         return True
 
+    def OnData(self, x, y, defResult):
```

Comment:
Why is it needed to override OnData ?

Reply:
I got an exception that it is not implemented and the application crashed

Reply:
It doesn't match [documentation](https://docs.wxpython.org/wx.DropTarget.html?highlight=ondata#wx.DropTarget.OnData).
Is that repeatable with latest release installer ? What version of WxPython are  you using ?